controllers/common/fetch.go

```diff
@@ -46,6 +46,35 @@ func FetchOperandRegistry(c client.Client, key types.NamespacedName) (*apiv1alph
 	return reg, nil
 }
 
+// FetchALLOperandRegistry lists the OperandRegistry instance with default value
+func FetchALLOperandRegistry(c client.Client, label map[string]string) (*apiv1alpha1.OperandRegistryList, error) {
```

Comment:
why not name the function the same as master branch PR one https://github.com/IBM/operand-deployment-lifecycle-manager/pull/609/files#diff-7b924b0b44ff7b9702a981217edb4f0fd0bb4ec3205ca3f5ebd54b895f9a0d32R75-R76?

Reply:
I want to keep the function name being consistent with other fetch functions in 1.4